lua/core/menu/home.lua

```diff
@@ -24,6 +24,7 @@ m.key = function(n,z)
     else
       norns.script.clear()
       _norns.free_engine()
+      params.reset()
```

Comment:
little late--- this throws an error, and i'm wondering what the original intention was?